scss/books-list.scss

```diff
@@ -1,3 +1,8 @@
+body {
+  height: 1200px;
+  background-color: $wild-sand;
+}
+
```

Comment:
I will put this in application.csss, because it seems it will be the same for all the application.

Reply:
Actually I hated to do this. I just did it to make the page scrollable, so the "sticky" effect in the nav bar can be seen. But as the view design doesn't actually has this, you'll see I had to remove this in a further PR. As I said, I just did it to make the page scrollable to show the navbar... I changed it as you suggest in `application.scss` in the last commit, but do you know a clearer solution to accomplish what I tried to do on this?

Reply:
What you did is good to test out the styles, but remember to delete if it's not the desired styles that will go to the final version of the app.